README.md

```diff
@@ -1,21 +1,90 @@
-Sensu-Plugins-gpg
-
 [ ![Build Status](https://travis-ci.org/sensu-plugins/sensu-plugins-gpg.svg?branch=master)](https://travis-ci.org/sensu-plugins/sensu-plugins-gpg)
 [![Gem Version](https://badge.fury.io/rb/sensu-plugins-gpg.svg)](http://badge.fury.io/rb/sensu-plugins-gpg)
 [![Code Climate](https://codeclimate.com/github/sensu-plugins/sensu-plugins-gpg/badges/gpa.svg)](https://codeclimate.com/github/sensu-plugins/sensu-plugins-gpg)
 [![Test Coverage](https://codeclimate.com/github/sensu-plugins/sensu-plugins-gpg/badges/coverage.svg)](https://codeclimate.com/github/sensu-plugins/sensu-plugins-gpg)
 [![Dependency Status](https://gemnasium.com/sensu-plugins/sensu-plugins-gpg.svg)](https://gemnasium.com/sensu-plugins/sensu-plugins-gpg)
 
-## Functionality
+# Sensu Plugins GPG
+
+This plugin is used for monitoring GPG keys.
 
 ## Files
- * bin/check-gpg-expiration.rb
 
-## Usage
+* bin/check-gpg-expiration.rb
+* bin/check-all-gpg-keys-for-expiry.rb
 
 ## Installation
```

Comment:
Or maybe remove the detailed installation instructions altogether since it's going to change in the future and there will be some new tool and I'd rather document it one place than 180+ places.

Reply:
agreed, good call. We should maybe link to a central doc in https://github.com/community that we link to that way its one modification. For now we should definitely remove it..

Reply:
I have removed the installation section.